src/mbgl/shaders/mtl/symbol_icon.cpp

```diff
@@ -3,23 +3,23 @@
 namespace mbgl {
 namespace shaders {
 
+const std::array<UniformBlockInfo, 5> ShaderSource<BuiltIn::SymbolIconShader, gfx::Backend::Type::Metal>::uniforms = {
+    UniformBlockInfo{true, true, sizeof(SymbolDynamicUBO), idSymbolDynamicUBO},
+    UniformBlockInfo{true, true, sizeof(SymbolDrawableUBO), idSymbolDrawableUBO},
+    UniformBlockInfo{true, false, sizeof(SymbolTilePropsUBO), idSymbolTilePropsUBO},
```

Comment:
`tileprops` ubo is not bound into fragment

Reply:
Fixed.